org.eclipse.gef.dot/src/org/eclipse/gef/dot/internal/language/htmllabel/DotHtmlLabelHelper.java

```diff
@@ -235,7 +235,10 @@ public static boolean isValidSiblings(List<HtmlContent> siblings) {
 			HtmlTag tag = content.getTag();
 			String text = content.getText();
 
-			if (tag != null && ("TABLE".equals(tag.getName().toUpperCase()))) {
+			if (tag != null && tag.getName() == null) {
+				continue;
```

Comment:
@miklossy This is needed for ca suggestion for an empty html like label tag to prevent NPE. Do yiu think this breaks sth else (as obviously an empty tag is illegal).